shared/desktop/yarn-helper/font.js

```diff
@@ -210,66 +111,108 @@ function updatedFonts() {
       }
     })
 
-  const grids = svgToGridMap()
-  let charCode = 0xe900
-
-  Object.keys(grids).forEach(gridSize => {
-    Object.keys(grids[gridSize]).forEach(name => {
-      const info = grids[gridSize][name]
-      icons[`iconfont-${info.name}`] = {
-        charCode,
-        gridSize: info.gridSize,
-        isFont: true,
-      }
-      charCode++
-    })
+  // Build constants for iconfont svgs
+  const svgFiles = getSvgFilenames()
+  console.log(svgFiles)
+  svgFiles.forEach(path => {
+    const match = path.match(iconfontRegex)
+    if (!match || match.length !== 4) return
+
+    const counter = Number(match[1])
```

Comment:
we can be more modern with es6 here
```
const [, counter, name, size] = match
```

Reply:
Changed this to just pull from `getSvgNames`